tiledb/sm/filesystem/s3.cc

```diff
@@ -1701,7 +1698,7 @@ Status S3::finish_flush_object(
   file_buffers_lck.unlock();
   tdb_delete(buff);
 
-  if (!outcome.IsSuccess()) {
+  if (!outcome.IsSuccess() || is_abort) {
     return LOG_STATUS(Status_S3Error(
         std::string("Failed to flush S3 object ") + uri.c_str() +
         outcome_error_message(outcome)));
```

Comment:
It looks like from the output shared in the PR description the word `Success` is being appended to the error string here when it's an AbortMultipartUpload request. Maybe only append with `outcome_error_message` when `is_abort` is false?

Reply:
done